### PR TITLE
424 add mask layer

### PR DESCRIPTION
**Pull request by anja-s**

### CODE REVIEW

File: oereb_client/views/index.py [hunk 1]
```diff
@@ -49,6 +49,9 @@ def validate_(self):
         if not isinstance(self.config_.get('availability'), dict):
             raise ConfigurationError('Missing "availability" configuration')
 
+        if not isinstance(self.config_.get('mask_surrounding'), dict):
```
Comment (michmuel) [inline oereb_client/views/index.py:52]: I think the config of masking of the surroundings should be optional. I guess some cantons will not use it.

**Review by michmuel — CHANGES_REQUESTED** [review]

File: oereb_client/views/index.py [hunk 2]
```diff
@@ -166,6 +169,7 @@ def get_config(self):
             'view': self.config_.get('view', {}),
             'base_layer': self.config_.get('base_layer', {}),
             'availability': self.config_.get('availability', {}),
+            'mask_surrounding': self.config_.get('mask_surrounding', {}),
```
Comment (kdeininger) [inline oereb_client/views/index.py:172]: You should remove the default value "{}". This way you could easily check if a mask layer is defined in the javascript code (mask_surrounding !== null).

File: oereb_client/static/src/component/map/map.js [hunk 3]
```diff
@@ -109,6 +110,20 @@ const OerebMap = function () {
...
   const [availabilityLayer] = useState(new LayerClass(availabilityConfig));
 
+  // Create mask_surrounding layer
+  const maskSurroundingConfig = {
+    preload: Infinity,
+    source: new SourceClass({
+      url: config.mask_surrounding.url
+    }),
+    opacity: 0.6,
+    zIndex: 10000
```
Comment (kdeininger) [inline oereb_client/static/src/component/map/map.js:120]: Use a higher zIndex (e.g. 40000) to display mask layer on top of all other layers.

**Review by kdeininger — CHANGES_REQUESTED** [review]
Please also add a configuration example in https://github.com/openoereb/oereb_client/blob/7861ab332eaa192fe37bebddaa3407855ce8471b/docs/src/Configuration.stories.mdx (see optional examples).

File: oereb_client/views/index.py (hunk 2 above)
Comment (anja-s) [reply oereb_client/views/index.py:178]: https://github.com/openoereb/oereb_client/blob/c131dd2015af83bba9c51bd382c9b4db7b55e9d9/oereb_client/views/index.py#L158-L164

Comment (anja-s) [reply oereb_client/views/index.py:178]: https://github.com/openoereb/oereb_client/blob/c131dd2015af83bba9c51bd382c9b4db7b55e9d9/oereb_client/views/index.py#L158-L164

Comment (anja-s) [reply oereb_client/views/index.py:178]: However, I also removed the default value "{}".

**Review by michmuel — APPROVED** [review]

**Review by kdeininger — APPROVED** [review]

**Comment by kdeininger** [comment]
Nice work :+1:

**Pull request merged**